ownCloud/AppDelegate.swift

```diff
@@ -80,8 +85,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 		OCExtensionManager.shared.addExtension(LinksAction.actionExtension)
 		OCExtensionManager.shared.addExtension(FavoriteAction.actionExtension)
 		OCExtensionManager.shared.addExtension(UnfavoriteAction.actionExtension)
-		// OCExtensionManager.shared.addExtension(ScanAction.actionExtension)
-
+		OCExtensionManager.shared.addExtension(ScanAction.actionExtension)
```

Comment:
`ScanAction` should be only available for >= iOS 13

Reply:
That's already dealt with in the `ScanAction` code, by returning `.none` for` applicablePosition` when not running under iOS 13:

https://github.com/owncloud/ios-app/blob/eb3e5160c014917c1f43cdc11c2b3a8e9a5f29fb/ownCloud/Client/Actions/Scanner/ScanAction.swift#L43

That way, this compatibility/implementation detail of `ScanAction` is fully contained in the action itself and doesn't leak to the `AppDelegate`.